src/App.js

```diff
@@ -1,5 +1,5 @@
 import './App.css';
-import Calculator from './components/Calculator';
+import Calculator from './Component/calculator';
```

Comment:
- [ ] according to the requirements the App.js component should be class-based, please adjust this before requesting another review.